topics/gradle-application-plugin.md

```diff
@@ -0,0 +1,67 @@
+[//]: # (title: Gradle Application plugin)
+
+<microformat>
+<p>
+<control>Sample project</control>: <a href="https://github.com/ktorio/ktor-gradle-sample/tree/main">ktor-gradle-sample</a>
+</p>
+</microformat>
+
+The Gradle [Application plugin](https://docs.gradle.org/current/userguide/application_plugin.html) provides the ability to package applications, including code dependencies and generated start scripts. In this topic, we'll show you how to package and run a Ktor application created in the [](Gradle.xml) topic.
+
+## Prerequisites {id="prerequisites"}
+Before starting this tutorial, do the following:
+* [Install IntelliJ IDEA](https://www.jetbrains.com/help/idea/installation-guide.html)
+* Clone the [ktor-gradle-sample](https://github.com/ktorio/ktor-gradle-sample) repository.
+
+
+## Apply the Application plugin and configure the main class {id="apply-plugin"}
+To package an application, you need to apply the Application plugin first:
+1. Open the [ktor-gradle-sample](https://github.com/ktorio/ktor-gradle-sample) project in IntelliJ IDEA.
+1. Make sure that the `build.gradle` file contains the following code:
+   ```groovy
+   ```
+   {src="https://raw.githubusercontent.com/ktorio/ktor-gradle-sample/main/build.gradle" lines="1,3-4,8,9-11"}
+    * The `id 'application'` line applies the Application plugin. 
```

Comment:
I am just curious, could we not duplicate content by marking some lines with numbers and refer them later?